examples/treegrid/treegrid-1.html

```diff
@@ -403,11 +537,11 @@ <h2>Javascript and CSS Source Code</h2>
     <ul>
       <li>
         CSS:
-        <a href="css/treegrid-1.css" type="tex/css">example_name.css</a>
+        <a href="css/treegrid-1.css" type="text/css">treegrid-1.css</a>
       </li>
       <li>
         Javascript:
-        <a href="js/treegrid-1.js" type="text/javascript">example_name.js</a>
+        <a href="js/treegrid-1.js" type="text/javascript">treegrid-1.js</a>
       </li>
     </ul>
   </section>
```

Comment:
There's a link at the end of the file back to the pattern than needs to be updated. Github doesn't let me comment on that line.